.github/workflows/codestyle.yml

```
@@ -0,0 +1,39 @@
+name: Codestyle
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
```

```suggestion
    branches:
      - "*"
    paths:
    - '**.php'
  pull_request:
    branches:
      - "*"
    paths:
    - '**.php'
```

Maybe skip this if no php files were modified?

---

Good point, I will also add composer.json the github actions and phpcs config file because they can influence the result.